corehq/apps/ota/views.py

```diff
@@ -63,8 +63,8 @@ def search(request, domain):
                  .case_type(case_type)
                  .is_closed(False)
                  .size(CASE_SEARCH_MAX_RESULTS))
-    config = CaseSearchConfig(domain=domain).config
-    fuzzies = config.get_fuzzy_properties_for_case_type(case_type)
+    config, _ = CaseSearchConfig.objects.get_or_create(domain=domain)
```

Comment:
If this is a valid assertion i'd be in favor of changing it to just a `get`. i always find it a little unsettling when requests/methods that look like gets make objects in the database